<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QMainWindow" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#lbl_ConnectionStatus[state = &quot;1&quot;]{
	background-color: rgb(51, 209, 122);
}
#lbl_ConnectionStatus[state = &quot;0&quot;]{
	background-color: rgb(224, 27, 36);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>10</y>
      <width>211</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="lst_Console"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>233</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="grp_ServerConnectionBox">
       <property name="title">
        <string>Server Connection</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_Port">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spb_Port">
          <property name="minimum">
           <number>1024</number>
          </property>
          <property name="maximum">
           <number>66356</number>
          </property>
          <property name="value">
           <number>12345</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn_StartServer">
          <property name="text">
           <string>Start Server</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbl_ConnectionStatus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>23</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius : 5px</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="state" stdset="0">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grp_MessageBox">
       <property name="title">
        <string>Message</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_Message">
          <property name="text">
           <string>Message</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ln_Message"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn_Send">
          <property name="text">
           <string>Send </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_Clear</sender>
   <signal>clicked()</signal>
   <receiver>lst_Console</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
